5-analysis/analysis-powder-diffraction.ipynb

```diff
@@ -0,0 +1,1665 @@
+    "quality of the fit numerically, we can look at the goodness-of-fit\n",
+    "χ² value and the reliability R-factors. The χ² value is a measure of how well\n",
+    "the calculated diffraction pattern matches the measured pattern, and it is\n",
+    "calculated as the sum of the squared differences between the measured and\n",
+    "calculated intensities, divided by the number of data points. Ideally, the\n",
+    "χ² value should be close to 1, indicating a good fit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b6b10e3a",
+   "metadata": {},
+   "source": [
+    "#### Visualize Fit Results\n",
+    "\n",
+    "After the fit is completed, we can plot the comparison between the\n",
+    "measured and calculated diffraction patterns again to see how well the fit\n",
+    "improved the agreement between the two. The calculated diffraction pattern\n",
+    "is now based on the refined parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "dbd9d48f",
+   "metadata": {},
+   "source": [
+    "project_1.plot_meas_vs_calc(expt_name=\"sim_si\")"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3728cc3d",
+   "metadata": {},
+   "source": [
+    "#### TOF vs d-spacing\n",
+    "\n",
+    "The diffraction pattern is typically analyzed and plotted in the\n",
+    "time-of-flight (TOF) axis, which represents the time it takes for neutrons\n",
+    "to travel from the sample to the detector. However, it is sometimes more\n",
+    "convenient to visualize the diffraction pattern in the d-spacing axis,\n",
+    "which represents the distance between planes in the crystal lattice. The\n",
+    "d-spacing can be calculated from the TOF values using the instrument\n",
+    "parameters. The `plot_meas_vs_calc` method of the project allows us to\n",
+    "plot the measured and calculated diffraction patterns in the d-spacing axis\n",
+    "by setting the `d_spacing` parameter to `True`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "ba29f44a",
+   "metadata": {},
+   "source": [
+    "project_1.plot_meas_vs_calc(expt_name=\"sim_si\", d_spacing=True)"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "372cfac2",
+   "metadata": {},
+   "source": [
+    "As you can see, the calculated diffraction pattern now matches the measured\n",
+    "pattern much more closely. Typically, additional parameters are included in the\n",
+    "refinement process to further improve the fit. However, we will stop here,\n",
+    "as the goal of this part of the tutorial is to demonstrate that the data reduction\n",
+    "and fitting process function correctly. The fit is not perfect, but it is\n",
+    "sufficient to show that the fitting process works and that the parameters\n",
+    "are being adjusted appropriately. The next part of the tutorial will be more advanced\n",
```

Comment:
Honestly, this is a stretch. The fit failed to reproduce the low-d peaks and the positions are wrong for many high-d peaks. Can this be improved by fitting additional parameters? If so, is there a reason you didn't do it here?